### ISSUE TITLE
‘TCGAbiolinksGUI’ is not available (for R version 3.3.2)

### ISSUE THREAD

**Issue by blyoung** [BioinformaticsFMRP/TCGAbiolinks#106]
Hi,
I am trying to install the TCGA/biolinks GUI and I get the following message below. Any advice on how I might fix this? I tried running this in both R and RStudio. 

Thanks,

Warning messages:
1: package ‘TCGAbiolinksGUI’ is not available (for R version 3.3.2)
2: In install.packages(update[instlib == l, "Package"], l, repos = repos, :
installation of package ‘RSQLite’ had non-zero exit status

**Comment by tiagochst** [comment]
It should be available for  R (>= 3.3.1).
Which commands are you using to install it?

You should be able to install it though Bioconductor (version 3.5)
```
source("https://bioconductor.org/biocLite.R")
biocLite("TCGAbiolinksGUI")
```

**Comment by blyoung** [comment]
Thanks Tiago for your support. I'm having to repeat all of my analysis because I was using the old cbioportal. I now want to strictly use TCGAbiolink, but I don't have R scripting skills and I find understanding the code to be challenging. So I really would like to use the GUI. Haveing said this, I am using the following code:

## try http:// if https:// URLs are not supported
source("https://bioconductor.org/biocLite.R")
biocLite("TCGAbiolinksGUI")

And I also tried http://

I ran it again after restarting Rstudio, and here is more of the message - 

installing to /Library/Frameworks/R.framework/Versions/3.3/Resources/library/dplyr/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** preparing package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (dplyr)

The downloaded source packages are in
	‘/private/var/folders/h6/4m85ngtd6293sfjl_5cc_f740000gn/T/Rtmp0uNGlw/downloaded_packages’
Warning message:
package ‘TCGAbiolinksGUI’ is not available (for R version 3.3.2) 



Kind Thanks,
Brandy

**Comment by DarioS** [comment]
TCGAbiolinksGUI first became available in Bioconductor version 3.5, which [requires R version 3.4](http://www.bioconductor.org/news/bioc_3_5_release/) to be installed. If you want to use TCGAbiolinksGUI with R version 3.3, you will need to manually download the software from the website and then use `install.packages` with `repos = NULL` as well as installing all of the dependencies of TCGAbiolinksGUI manually. It's actually much easier to upgrade your version of R, so that you can use `biocLite`.

It's weird that both [release](http://bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html) and [development](http://bioconductor.org/packages/devel/bioc/html/TCGAbiolinksGUI.html) versions have version number 1.1.0. Perhaps someone made a mistake when committing changes to the release branch. The release version number should have the format 1.0.x.
